int32 resource_number
nav_msgs/Path[] resource_plan